<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Helpers</class>
 <widget class="QDialog" name="Helpers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Helper Editor</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>531</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
   </property>
  </widget>
  <widget class="QListWidget" name="helpersList">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>40</y>
     <width>99</width>
     <height>151</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="removeHelper">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>70</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/list-remove.svgz</normaloff>:/icon/list-remove.svgz</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="addHelper">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>40</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/list-add.svgz</normaloff>:/icon/list-add.svgz</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>21</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Helpers list:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>16</width>
     <height>151</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="helperCmdLine">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="helperGranularity">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>43</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>130</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Granularity: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>40</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Helper's Name: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Command Line: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="helperName">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>100</y>
     <width>107</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>File's Extension(s): </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="helperFileExt">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <zorder>helperCmdLine</zorder>
  <zorder>helperGranularity</zorder>
  <zorder>label_5</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>helperName</zorder>
  <zorder>label_4</zorder>
  <zorder>buttonBox</zorder>
  <zorder>helpersList</zorder>
  <zorder>removeHelper</zorder>
  <zorder>addHelper</zorder>
  <zorder>label</zorder>
  <zorder>line</zorder>
  <zorder>helperFileExt</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Helpers</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Helpers</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
